build.sbt

```diff
@@ -956,6 +956,9 @@ lazy val sql = (project in component("sql")).
     name := "nussknacker-sql",
     libraryDependencies ++= Seq(
       "com.zaxxer" % "HikariCP" % "4.0.3",
+//      It won't run on Java 16 as Hikari will fail while trying to load IgniteJdbcThinDriver https://issues.apache.org/jira/browse/IGNITE-14888
+      "org.apache.ignite" % "ignite-core" % "2.10.0",
```

Comment:
let's have it as test/optional

Reply:
+1 ... and add notice in documentation that to use it you should add this extra libs to `modelConfig.classPath`